modules/probe.sh

```diff
@@ -94,6 +94,50 @@ probe_debian_arch_name() {
   echo "${deb_arch}"
 }
 
+# For x86 distros like e.g. Ubuntu try to find grubx64.efi
+# file and return it's path otherwise return empty path
+#
+# No input parameters and these work on the assumption that boot and root parts
+# are mounted at work/boot
+find_grub_efi_path() {
+  grub_path=$(sudo find work/boot -name grubx64.efi)
+  echo "${grub_path}"
```

Comment:
`grub_path` is entirely redundant here, you can just output `find` directly.

Reply:
:+1: updated and added changelogs